static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,243 @@
 # fetch
 
-Fetches data files from the remote storage to the local cache.
+Get files that are under DVC control from remote storage into the local cache.
+Fetching means downloading from the
+[remote](https://dvc.org/doc/commands-reference/remote#description) unless the
```

Comment:
no need for the absolute path here, a relative path like `/doc/commands-reference/remote` should do.

Reply:
I'm actually not sure if we need the second sentence at all, let's just mode the link to the first sentence and keep only it

Reply:
Done